README.md

```diff
@@ -1,139 +1,167 @@
+  oc adm policy add-scc-to-user anyuid system:serviceaccount:myproject:default
+  oc adm policy add-scc-to-user privileged system:serviceaccount:myproject:default
+  ```
 
-where the GIT token should have `public_repo`, `read:org`, and `admin:repo_hook` permissions. 
+- Deploy the Istio control plane.
+  If you also want to install the Fabric8 Launcher, edit the `istio-installation.yaml` file.
 
-The more complete version may look like this:
-https://github.com/Maistra/openshift-ansible/blob/maistra-0.9.0/istio/istio-installation-full.yaml
+  ```bash
+  oc apply -f istio-installation.yaml
+  ```
 
-but the one above is sufficient for testing the boosters. Note, setting `istio.authentication` to `true` enables MTLS.
+- Wait for the Istio control plane deployment to finish. Run
 
-- Verify the Istio Control Plane and Launcher deployments:
+  ```bash
+  oc get pods -n istio-system -w
+  ```
 
-https://github.com/Maistra/openshift-ansible/blob/maistra-0.9.0/istio/Installation.md#verifying-the-istio-control-plane
+  or open the OpenShift web console and wait until everything settles down.
+  In the end, running `oc get pods -n istio-system` should show something like this:
 
-Now proceed to testing the booster.
+  ```
+  NAME                                          READY     STATUS      RESTARTS   AGE
+  elasticsearch-0                               1/1       Running     0          1m
+  grafana-74b5796d94-796tc                      1/1       Running     0          1m
+  istio-citadel-5d595fd7d7-w9lqd                1/1       Running     0          2m
+  istio-egressgateway-594684895-tp9w7           1/1       Running     0          2m
+  istio-galley-699f48cb-xbxf4                   1/1       Running     0          2m
+  istio-ingressgateway-568cd96f58-gb2fm         1/1       Running     0          2m
+  istio-pilot-99fc457ff-h59d5                   2/2       Running     0          2m
+  istio-policy-85d8ffdb98-nn8kw                 2/2       Running     3          2m
+  istio-sidecar-injector-5d5bf88c78-c2hks       1/1       Running     0          2m
+  istio-telemetry-5c75d844cf-s76x9              2/2       Running     3          2m
+  jaeger-agent-lbcnv                            1/1       Running     0          1m
+  jaeger-collector-86d57594d5-7lktp             1/1       Running     1          1m
+  jaeger-query-f96b97b75-bm2sw                  1/1       Running     1          1m
+  kiali-59fd54974c-4c9xz                        1/1       Running     0          1m
+  openshift-ansible-istio-installer-job-ddvdr   0/1       Completed   0          4m
+  prometheus-75b849445c-8n8rv                   1/1       Running     0          2m
+  ```
 
-## Launcher Flow Setup
+- Remember the URL of the Istio ingress gateway:
 
-If the Booster is installed through the Launcher and the Continuous Delivery flow, no additional steps are necessary.
+  ```bash
+  ISTIO_INGRESS=$(oc get route istio-ingressgateway -n istio-system -o jsonpath='{.spec.host}')
```

Comment:
This requires the `admin` user ? If yes then I'm not sure if it makes it simpler to use a 'developer' user as well, if one has to switch between 'admin' and 'developer'

Reply:
It does, because it gets a resource from the `istio-system` namespace. That's why I put it into the "preparing Istio" section, in which we instruct the user to be an admin.

Reply:
Thanks

Reply:
Sorry, there is still a `developer` user in your changes

Reply:
This is probably cleaner though, the `admin' prepares the namespace, and the real `developer' works with it

Reply:
Yea, there's still `developer`, but the env var `ISTIO_INGRESS` is set by the admin. I couldn't think of a better way. Well currently the `istio-system` and `istio-ingressgateway` names are hardcoded, so perhaps we could hardcode them as well...